spec/unit/application/apply_spec.rb

```diff
@@ -3,7 +3,6 @@
 
 require 'puppet/application/apply'
 require 'puppet/file_bucket/dipper'
-require 'puppet/configurer'
```

Comment:
Puppet::Configurer is used on line 202

Reply:
I must have accidentally removed this.  Wonder why specs are still happy ?

Reply:
Most likely `Configurer` was required previous to this file by a different test. I imagine running this file in isolation (or in a bad order) would fail.

Reply:
Just added it back in.